sdk/sdk-client/src/utils/PositionUtils.ts

```diff
@@ -49,6 +49,6 @@ export class PositionUtils {
     const liquidationPrice = debtValue.div(
       collateralAmount.times(liquidationThreshold.toProportion()),
     )
-    return liquidationPrice.toString()
+    return liquidationPrice.times(debtPriceInUsd).toString()
```

Comment:
I think we could refactor this to use the SDK types like Price, TokenAmount and Percentage (I see we already use Percentage), but it is good for now